mesheryctl/internal/cli/root/system/logs.go

```diff
@@ -42,10 +41,10 @@ var logsCmd = &cobra.Command{
 
 		log.Info("Starting Meshery logging...")
 
-		if _, err := os.Stat(utils.DockerComposeFile); os.IsNotExist(err) {
-			if err := utils.DownloadFile(utils.DockerComposeFile, fileURL); err != nil {
-				return errors.Wrapf(err, utils.SystemError(fmt.Sprintf("failed to download %s file from %s", utils.DockerComposeFile, fileURL)))
-			}
+		// download(don't force) the docker-compose.yaml file corresponding to the current version
+		_, currCtx, err := utils.GetCurrentContext(tempContext)
+		if _, err := utils.DownloadDockerComposeFile(currCtx, false); err != nil {
```

Comment:
skip the download of docker-compose file if it already exists (since meshery must be already running, the file should be correct?)

Reply:
It should be correct. @anirudhjain75 @leecalcote This can be done right?

Reply:
Right. If the user wants to see Meshery logs and there is no docker-compose file, then we should invoke a central function that does something like:

1. Inform the user of the missing deployment file.
2. Ask if they would like mesheryctl to retrieve a new deployment file (or...)
3. Ask if they would like mesheryctl to generate a new deployment file (assuming that mesheryctl has this ability at some point).